include/arch/arm/armv/armv8-a/64/armv/vcpu.h

```diff
@@ -630,9 +630,12 @@ static inline void armv_vcpu_boot_init(void)
 #endif
 }
 
-static inline void armv_vcpu_save(vcpu_t *vcpu, UNUSED bool_t active)
+static inline void armv_vcpu_save(vcpu_t *vcpu, bool_t active)
 {
-    vcpu_save_reg_range(vcpu, seL4_VCPUReg_TTBR0, seL4_VCPUReg_SPSR_EL1);
+    if (active) {
+        vcpu_save_reg(vcpu, seL4_VCPUReg_CPACR);
+    }
```

Comment:
Please add the same comment that `vcpu_save` has for SCTLR, as it's the same, except 64-bit only:
```suggestion
    /* If we aren't active then this state already got stored when we were disabled */
    if (active) {
        vcpu_save_reg(vcpu, seL4_VCPUReg_CPACR);
    }
```

Reply:
Done